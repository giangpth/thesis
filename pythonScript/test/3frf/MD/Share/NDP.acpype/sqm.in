Run semi-empirical minimization
 &qmmm
    qm_theory='AM1', grms_tol=0.0005,
 scfconv=1.d-10, ndiis_attempts=700,   qmcharge=0,
 /
  15    PA       24.0140       21.5500       42.3840 
   8   O1A       25.2110       20.6920       42.2310 
   8   O2A       22.8340       21.0820       41.6300 
   8   O5B       24.3730       23.0380       41.9430 
   6   C5B       23.4270       24.0530       42.1470 
   6   C4B       23.9400       25.2300       41.3540 
   8   O4B       23.9740       24.9070       39.9770 
   6   C3B       23.0920       26.4810       41.4620 
   8   O3B       23.3040       27.1710       42.6660 
   6   C2B       23.6270       27.2000       40.2550 
   8   O2B       24.9370       27.6300       40.5360 
   6   C1B       23.7170       26.0840       39.2300 
   7   N9A       22.4530       25.9470       38.4960 
   6   C8A       21.5270       24.9530       38.6680 
   7   N7A       20.5070       25.1690       37.8110 
   6   C5A       20.7700       26.2860       37.0950 
   6   C6A       20.0650       26.9440       36.0970 
   7   N6A       18.8930       26.4770       35.6830 
   7   N1A       20.5890       28.0880       35.5360 
   6   C2A       21.8080       28.5700       35.9730 
   7   N3A       22.5030       27.9130       36.9680 
   6   C4A       21.9920       26.7870       37.5200 
   8    O3       23.7070       21.7690       43.9420 
  15    PN       22.7030       21.0560       44.9660 
   8   O1N       21.3440       21.1780       44.4000 
   8   O2N       23.0020       21.5710       46.3210 
   8   O5D       23.1720       19.5200       44.9260 
   6   C5D       24.4160       19.1890       45.4920 
   6   C4D       24.2800       17.9200       46.3110 
   8   O4D       24.1650       16.8080       45.4470 
   6   C3D       25.4990       17.6720       47.1760 
   8   O3D       25.0860       17.2540       48.4530 
   6   C2D       26.2380       16.5580       46.4790 
   8   O2D       26.9660       15.7880       47.4030 
   6   C1D       25.0860       15.8120       45.8320 
   7   N1N       25.4240       15.0260       44.6380 
   6   C2N       25.1800       13.6880       44.6770 
   6   C3N       25.4570       12.9020       43.5760 
   6   C7N       25.0380       11.4680       43.6110 
   8   O7N       24.7770       10.7960       42.4020 
   7   N7N       24.9300       10.8840       44.8050 
   6   C4N       25.9760       13.4790       42.4210 
   6   C5N       26.2100       14.8490       42.3880 
   6   C6N       25.9230       15.6180       43.5080 
  15   P2B       25.2970       29.1340       40.9890 
   8   O1X       24.5570       30.0790       40.0710 
   8   O2X       26.7970       29.2850       40.8420 
   8   O3X       24.9110       29.3420       42.4360 
   1  H5B1       22.4480       23.7430       41.7820 
   1  H5B2       23.3710       24.3080       43.2050 
   1   H4B       24.9520       25.4620       41.6850 
   1   H3B       22.0360       26.2440       41.3340 
   1   H2B       22.9790       28.0150       39.9310 
   1   H1B       24.5360       26.2760       38.5360 
   1   H8A       21.6000       24.1350       39.3700 
   1   H2A       22.2190       29.4660       35.5330 
   1  H5D1       25.1460       19.0320       44.6980 
   1  H5D2       24.7500       20.0030       46.1360 
   1   H4D       23.3930       17.9860       46.9410 
   1   H3D       26.1180       18.5670       47.2410 
   1   H2D       26.9030       16.9630       45.7160 
   1   H1D       24.6210       15.1600       46.5710 
   1   H2N       24.7690       13.2400       45.5700 
   1   H4N       26.1950       12.8680       41.5580 
   1   H5N       26.6120       15.3110       41.4990 
   1   H6N       26.0930       16.6840       43.4880 
   1  H7N1       25.1290       11.4100       45.6440 
   1  H7N2       24.6490        9.9160       44.8700 
   1  HO3B       23.8580       27.9380       42.5030 
   1  H6A1       18.5100       25.6380       36.0950 
   1  H6A2       18.3850       26.9610       34.9560 
   1  HO3D       25.8570       17.0960       49.0030 
   1  HO2D       27.4290       15.0850       46.9410 
   1     H       23.0720       20.9660       40.6970 
   1    H9       19.2290       26.2770       36.0780 
   1   H10       23.3700       28.2630       37.2760 
   1   H11       22.7300       22.5000       46.3810 
   1   H12       26.1990       12.8790       41.5640 
   1   H13       27.0460       29.1520       39.9140 
   1   H14       23.9510       29.2460       42.5320 

