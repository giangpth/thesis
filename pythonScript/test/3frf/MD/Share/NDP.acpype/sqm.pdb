ATOM      1  PA  NDP   301      25.035  21.725  42.609  1.00  0.00           P
ATOM      2  O1A NDP   301      26.051  20.703  42.505  1.00  0.00           O
ATOM      3  O2A NDP   301      24.076  21.779  41.345  1.00  0.00           O
ATOM      4  O5B NDP   301      25.447  23.232  42.882  1.00  0.00           O
ATOM      5  C5B NDP   301      24.572  24.298  42.574  1.00  0.00           C
ATOM      6  C4B NDP   301      25.298  25.260  41.629  1.00  0.00           C
ATOM      7  O4B NDP   301      25.097  24.856  40.269  1.00  0.00           O
ATOM      8  C3B NDP   301      24.824  26.716  41.788  1.00  0.00           C
ATOM      9  O3B NDP   301      25.885  27.559  42.168  1.00  0.00           O
ATOM     10  C2B NDP   301      24.203  27.106  40.424  1.00  0.00           C
ATOM     11  O2B NDP   301      24.967  28.119  39.822  1.00  0.00           O
ATOM     12  C1B NDP   301      24.330  25.820  39.536  1.00  0.00           C
ATOM     13  N9A NDP   301      23.062  25.268  39.137  1.00  0.00           N
ATOM     14  C8A NDP   301      22.497  24.019  39.440  1.00  0.00           C
ATOM     15  N7A NDP   301      21.273  23.851  38.883  1.00  0.00           N
ATOM     16  C5A NDP   301      21.026  25.030  38.183  1.00  0.00           C
ATOM     17  C6A NDP   301      19.888  25.495  37.436  1.00  0.00           C
ATOM     18  N6A NDP   301      18.689  24.692  37.441  1.00  0.00           N
ATOM     19  N1A NDP   301      19.833  26.821  37.022  1.00  0.00           N
ATOM     20  C2A NDP   301      20.893  27.566  37.177  1.00  0.00           C
ATOM     21  N3A NDP   301      22.182  27.238  37.761  1.00  0.00           N
ATOM     22  C4A NDP   301      22.128  25.929  38.315  1.00  0.00           C
ATOM     23  O3  NDP   301      23.981  21.475  43.799  1.00  0.00           O
ATOM     24  PN  NDP   301      22.960  21.247  45.020  1.00  0.00           P
ATOM     25  O1N NDP   301      22.062  22.348  45.277  1.00  0.00           O
ATOM     26  O2N NDP   301      23.975  20.980  46.217  1.00  0.00           O
ATOM     27  O5D NDP   301      22.401  19.795  44.705  1.00  0.00           O
ATOM     28  C5D NDP   301      23.284  18.694  44.652  1.00  0.00           C
ATOM     29  C4D NDP   301      22.889  17.668  45.721  1.00  0.00           C
ATOM     30  O4D NDP   301      23.053  16.357  45.174  1.00  0.00           O
ATOM     31  C3D NDP   301      23.746  17.763  46.999  1.00  0.00           C
ATOM     32  O3D NDP   301      22.859  17.694  48.096  1.00  0.00           O
ATOM     33  C2D NDP   301      24.727  16.572  46.910  1.00  0.00           C
ATOM     34  O2D NDP   301      24.680  15.865  48.132  1.00  0.00           O
ATOM     35  C1D NDP   301      24.187  15.679  45.749  1.00  0.00           C
ATOM     36  N1N NDP   301      25.194  15.341  44.756  1.00  0.00           N
ATOM     37  C2N NDP   301      25.009  14.152  44.034  1.00  0.00           C
ATOM     38  C3N NDP   301      25.648  13.894  42.866  1.00  0.00           C
ATOM     39  C7N NDP   301      25.472  12.617  42.133  1.00  0.00           C
ATOM     40  O7N NDP   301      25.392  12.569  40.888  1.00  0.00           O
ATOM     41  N7N NDP   301      25.386  11.451  42.865  1.00  0.00           N
ATOM     42  C4N NDP   301      26.534  14.895  42.206  1.00  0.00           C
ATOM     43  C5N NDP   301      26.498  16.204  42.901  1.00  0.00           C
ATOM     44  C6N NDP   301      25.850  16.387  44.069  1.00  0.00           C
ATOM     45  P2B NDP   301      24.265  29.551  39.563  1.00  0.00           P
ATOM     46  O1X NDP   301      23.094  29.829  40.370  1.00  0.00           O
ATOM     47  O2X NDP   301      24.299  29.649  37.982  1.00  0.00           O
ATOM     48  O3X NDP   301      25.499  30.517  39.848  1.00  0.00           O
ATOM     49 H5B1 NDP   301      23.630  23.902  42.108  1.00  0.00           H
ATOM     50 H5B2 NDP   301      24.345  24.828  43.542  1.00  0.00           H
ATOM     51  H4B NDP   301      26.409  25.178  41.802  1.00  0.00           H
ATOM     52  H3B NDP   301      24.066  26.828  42.611  1.00  0.00           H
ATOM     53  H2B NDP   301      23.136  27.450  40.547  1.00  0.00           H
ATOM     54  H1B NDP   301      24.938  26.046  38.606  1.00  0.00           H
ATOM     55  H8A NDP   301      23.028  23.275  40.047  1.00  0.00           H
ATOM     56  H2A NDP   301      20.839  28.622  36.853  1.00  0.00           H
ATOM     57 H5D1 NDP   301      23.161  18.217  43.637  1.00  0.00           H
ATOM     58 H5D2 NDP   301      24.340  19.050  44.788  1.00  0.00           H
ATOM     59  H4D NDP   301      21.793  17.763  45.962  1.00  0.00           H
ATOM     60  H3D NDP   301      24.316  18.732  47.052  1.00  0.00           H
ATOM     61  H2D NDP   301      25.771  16.938  46.717  1.00  0.00           H
ATOM     62  H1D NDP   301      23.768  14.717  46.174  1.00  0.00           H
ATOM     63  H2N NDP   301      24.320  13.424  44.502  1.00  0.00           H
ATOM     64  H4N NDP   301      26.218  15.015  41.131  1.00  0.00           H
ATOM     65  H5N NDP   301      27.031  17.038  42.422  1.00  0.00           H
ATOM     66  H6N NDP   301      25.815  17.369  44.574  1.00  0.00           H
ATOM     67 H7N1 NDP   301      25.596  11.445  43.828  1.00  0.00           H
ATOM     68 H7N2 NDP   301      25.336  10.586  42.385  1.00  0.00           H
ATOM     69 HO3B NDP   301      26.517  27.587  41.435  1.00  0.00           H
ATOM     70 H6A1 NDP   301      18.977  23.717  37.344  1.00  0.00           H
ATOM     71 H6A2 NDP   301      18.086  24.959  36.663  1.00  0.00           H
ATOM     72 HO3D NDP   301      23.359  17.393  48.866  1.00  0.00           H
ATOM     73 HO2D NDP   301      25.390  15.211  48.105  1.00  0.00           H
ATOM     74  H   NDP   301      24.156  21.089  40.690  1.00  0.00           H
ATOM     75  H9  NDP   301      18.165  24.796  38.317  1.00  0.00           H
ATOM     76  H10 NDP   301      22.883  27.266  37.031  1.00  0.00           H
ATOM     77  H11 NDP   301      23.993  21.613  46.931  1.00  0.00           H
ATOM     78  H12 NDP   301      27.591  14.507  42.191  1.00  0.00           H
ATOM     79  H13 NDP   301      25.142  29.882  37.596  1.00  0.00           H
ATOM     80  H14 NDP   301      25.537  30.966  40.688  1.00  0.00           H
