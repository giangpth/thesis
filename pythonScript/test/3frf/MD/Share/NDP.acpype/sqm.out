            --------------------------------------------------------
                             AMBER SQM VERSION 19
 
                                     By
              Ross C. Walker, Michael F. Crowley, Scott Brozell,
                         Tim Giese, Andreas W. Goetz,
                        Tai-Sung Lee and David A. Case
 
            --------------------------------------------------------
 

--------------------------------------------------------------------------------
  QM CALCULATION INFO
--------------------------------------------------------------------------------

| QMMM: Citation for AMBER QMMM Run:
| QMMM: R.C. Walker, M.F. Crowley and D.A. Case, J. COMP. CHEM. 29:1019, 2008

QMMM: SINGLET STATE CALCULATION
QMMM: RHF CALCULATION, NO. OF DOUBLY OCCUPIED LEVELS =134

| QMMM: *** Selected Hamiltonian *** 
| QMMM: AM1         

| QMMM: *** Parameter sets in use ***
| QMMM: P : M.J.S.DEWAR et al. THEOCHEM, 187,1, (1989)
| QMMM: O : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: C : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: N : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)
| QMMM: H : M.J.S.DEWAR et al. JACS, 107, 3902, (1985)

| QMMM: *** SCF convergence criteria ***
| QMMM: Energy change                :  0.1D-09 kcal/mol
| QMMM: Error matrix |FP-PF|         :  0.1D+00 au
| QMMM: Density matrix change        :  0.5D-06
| QMMM: Maximum number of SCF cycles :     1000

| QMMM: *** Diagonalization Routine Information ***
| QMMM: Pseudo diagonalizations are allowed.
| QMMM: Auto diagonalization routine selection is in use.
| QMMM:
| QMMM: Timing diagonalization routines:
| QMMM:                              norbs =      224
| QMMM:    diag iterations used for timing =        4
| QMMM:
| QMMM:              Internal diag routine = 0.170889 seconds
| QMMM:                 Dspev diag routine = 0.195766 seconds
| QMMM:                Dspevd diag routine = 0.058953 seconds
| QMMM:                Dspevx diag routine = 0.267320 seconds
| QMMM:                 Dsyev diag routine = 0.213707 seconds
| QMMM:                Dsyevd diag routine = 0.050120 seconds
| QMMM:                Dsyevr diag routine = 0.058461 seconds
| QMMM:
| QMMM:                Pseudo diag routine = 0.080216 seconds
| QMMM: Pseudo diagonalization appears to be slower than regular
| QMMM: diagonalization. Setting pseudo_diag=0 for optimum performance.
| QMMM:
| QMMM: Using dsyevd routine (diag_routine=6).

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      P       24.0140   21.5500   42.3840
  QMMM:     2        2      O       25.2110   20.6920   42.2310
  QMMM:     3        3      O       22.8340   21.0820   41.6300
  QMMM:     4        4      O       24.3730   23.0380   41.9430
  QMMM:     5        5      C       23.4270   24.0530   42.1470
  QMMM:     6        6      C       23.9400   25.2300   41.3540
  QMMM:     7        7      O       23.9740   24.9070   39.9770
  QMMM:     8        8      C       23.0920   26.4810   41.4620
  QMMM:     9        9      O       23.3040   27.1710   42.6660
  QMMM:    10       10      C       23.6270   27.2000   40.2550
  QMMM:    11       11      O       24.9370   27.6300   40.5360
  QMMM:    12       12      C       23.7170   26.0840   39.2300
  QMMM:    13       13      N       22.4530   25.9470   38.4960
  QMMM:    14       14      C       21.5270   24.9530   38.6680
  QMMM:    15       15      N       20.5070   25.1690   37.8110
  QMMM:    16       16      C       20.7700   26.2860   37.0950
  QMMM:    17       17      C       20.0650   26.9440   36.0970
  QMMM:    18       18      N       18.8930   26.4770   35.6830
  QMMM:    19       19      N       20.5890   28.0880   35.5360
  QMMM:    20       20      C       21.8080   28.5700   35.9730
  QMMM:    21       21      N       22.5030   27.9130   36.9680
  QMMM:    22       22      C       21.9920   26.7870   37.5200
  QMMM:    23       23      O       23.7070   21.7690   43.9420
  QMMM:    24       24      P       22.7030   21.0560   44.9660
  QMMM:    25       25      O       21.3440   21.1780   44.4000
  QMMM:    26       26      O       23.0020   21.5710   46.3210
  QMMM:    27       27      O       23.1720   19.5200   44.9260
  QMMM:    28       28      C       24.4160   19.1890   45.4920
  QMMM:    29       29      C       24.2800   17.9200   46.3110
  QMMM:    30       30      O       24.1650   16.8080   45.4470
  QMMM:    31       31      C       25.4990   17.6720   47.1760
  QMMM:    32       32      O       25.0860   17.2540   48.4530
  QMMM:    33       33      C       26.2380   16.5580   46.4790
  QMMM:    34       34      O       26.9660   15.7880   47.4030
  QMMM:    35       35      C       25.0860   15.8120   45.8320
  QMMM:    36       36      N       25.4240   15.0260   44.6380
  QMMM:    37       37      C       25.1800   13.6880   44.6770
  QMMM:    38       38      C       25.4570   12.9020   43.5760
  QMMM:    39       39      C       25.0380   11.4680   43.6110
  QMMM:    40       40      O       24.7770   10.7960   42.4020
  QMMM:    41       41      N       24.9300   10.8840   44.8050
  QMMM:    42       42      C       25.9760   13.4790   42.4210
  QMMM:    43       43      C       26.2100   14.8490   42.3880
  QMMM:    44       44      C       25.9230   15.6180   43.5080
  QMMM:    45       45      P       25.2970   29.1340   40.9890
  QMMM:    46       46      O       24.5570   30.0790   40.0710
  QMMM:    47       47      O       26.7970   29.2850   40.8420
  QMMM:    48       48      O       24.9110   29.3420   42.4360
  QMMM:    49       49      H       22.4480   23.7430   41.7820
  QMMM:    50       50      H       23.3710   24.3080   43.2050
  QMMM:    51       51      H       24.9520   25.4620   41.6850
  QMMM:    52       52      H       22.0360   26.2440   41.3340
  QMMM:    53       53      H       22.9790   28.0150   39.9310
  QMMM:    54       54      H       24.5360   26.2760   38.5360
  QMMM:    55       55      H       21.6000   24.1350   39.3700
  QMMM:    56       56      H       22.2190   29.4660   35.5330
  QMMM:    57       57      H       25.1460   19.0320   44.6980
  QMMM:    58       58      H       24.7500   20.0030   46.1360
  QMMM:    59       59      H       23.3930   17.9860   46.9410
  QMMM:    60       60      H       26.1180   18.5670   47.2410
  QMMM:    61       61      H       26.9030   16.9630   45.7160
  QMMM:    62       62      H       24.6210   15.1600   46.5710
  QMMM:    63       63      H       24.7690   13.2400   45.5700
  QMMM:    64       64      H       26.1950   12.8680   41.5580
  QMMM:    65       65      H       26.6120   15.3110   41.4990
  QMMM:    66       66      H       26.0930   16.6840   43.4880
  QMMM:    67       67      H       25.1290   11.4100   45.6440
  QMMM:    68       68      H       24.6490    9.9160   44.8700
  QMMM:    69       69      H       23.8580   27.9380   42.5030
  QMMM:    70       70      H       18.5100   25.6380   36.0950
  QMMM:    71       71      H       18.3850   26.9610   34.9560
  QMMM:    72       72      H       25.8570   17.0960   49.0030
  QMMM:    73       73      H       27.4290   15.0850   46.9410
  QMMM:    74       74      H       23.0720   20.9660   40.6970
  QMMM:    75       75      H       19.2290   26.2770   36.0780
  QMMM:    76       76      H       23.3700   28.2630   37.2760
  QMMM:    77       77      H       22.7300   22.5000   46.3810
  QMMM:    78       78      H       26.1990   12.8790   41.5640
  QMMM:    79       79      H       27.0460   29.1520   39.9140
  QMMM:    80       80      H       23.9510   29.2460   42.5320

--------------------------------------------------------------------------------
  RESULTS
--------------------------------------------------------------------------------

      iter         sqm energy              rms gradient
      ----    -------------------    -----------------------
xmin    10     -810.6302 kcal/mol        6.2140 kcal/(mol*A)
xmin    20     -843.5398 kcal/mol        1.0817 kcal/(mol*A)
xmin    30     -847.1005 kcal/mol        1.1552 kcal/(mol*A)
xmin    40     -848.7982 kcal/mol        0.7047 kcal/(mol*A)
xmin    50     -850.0920 kcal/mol        0.2189 kcal/(mol*A)
xmin    60     -851.1299 kcal/mol        0.2850 kcal/(mol*A)
xmin    70     -852.3050 kcal/mol        0.7362 kcal/(mol*A)
xmin    80     -853.2450 kcal/mol        0.1087 kcal/(mol*A)
xmin    90     -853.5681 kcal/mol        0.2478 kcal/(mol*A)
xmin   100     -853.6681 kcal/mol        0.0721 kcal/(mol*A)
xmin   110     -853.7197 kcal/mol        0.2412 kcal/(mol*A)
xmin   120     -853.8152 kcal/mol        0.1694 kcal/(mol*A)
xmin   130     -853.8610 kcal/mol        0.1148 kcal/(mol*A)
xmin   140     -853.8944 kcal/mol        0.1371 kcal/(mol*A)
xmin   150     -853.9393 kcal/mol        0.0317 kcal/(mol*A)
xmin   160     -853.9524 kcal/mol        0.0268 kcal/(mol*A)
xmin   170     -853.9573 kcal/mol        0.0206 kcal/(mol*A)
xmin   180     -853.9612 kcal/mol        0.0228 kcal/(mol*A)
xmin   190     -853.9662 kcal/mol        0.0195 kcal/(mol*A)
xmin   200     -853.9685 kcal/mol        0.0278 kcal/(mol*A)
xmin   210     -853.9713 kcal/mol        0.0147 kcal/(mol*A)
xmin   220     -853.9733 kcal/mol        0.0062 kcal/(mol*A)
xmin   230     -853.9736 kcal/mol        0.0068 kcal/(mol*A)
xmin   240     -853.9739 kcal/mol        0.0051 kcal/(mol*A)
xmin   250     -853.9743 kcal/mol        0.0107 kcal/(mol*A)
xmin   260     -853.9746 kcal/mol        0.0018 kcal/(mol*A)
xmin   270     -853.9747 kcal/mol        0.0056 kcal/(mol*A)
xmin   280     -853.9747 kcal/mol        0.0010 kcal/(mol*A)
xmin   290     -853.9747 kcal/mol        0.0015 kcal/(mol*A)
xmin   300     -853.9748 kcal/mol        0.0021 kcal/(mol*A)
xmin   310     -853.9748 kcal/mol        0.0021 kcal/(mol*A)
xmin   320     -853.9748 kcal/mol        0.0017 kcal/(mol*A)
xmin   330     -853.9748 kcal/mol        0.0006 kcal/(mol*A)
  ... geometry converged !
 
 Final MO eigenvalues (eV):
 -42.0722  -42.0207  -40.7413  -40.1970  -39.4145  -39.3171  -38.7641  -38.7352
 -38.4354  -38.1021  -37.9702  -37.7687  -37.6980  -37.6051  -37.3680  -37.0401
 -36.9848  -36.7989  -35.0671  -34.9949  -34.8101  -34.3032  -34.0955  -33.9308
 -32.4149  -32.0695  -31.9956  -31.6372  -30.2979  -30.1693  -28.7436  -27.1506
 -26.6347  -25.6483  -24.6958  -24.4328  -24.0511  -23.4924  -23.3564  -23.2145
 -22.6739  -22.6416  -21.4408  -21.4267  -21.2862  -21.1290  -20.1779  -19.9445
 -19.8411  -19.7659  -19.3818  -19.2655  -19.1852  -19.0440  -18.9802  -18.9121
 -18.7816  -18.5773  -18.0843  -17.9656  -17.7764  -17.7189  -17.6280  -17.4404
 -17.4103  -17.1667  -16.9954  -16.9406  -16.8700  -16.7359  -16.7092  -16.6438
 -16.4419  -16.0814  -15.9190  -15.8504  -15.8119  -15.7056  -15.4979  -15.4472
 -15.3758  -15.2254  -15.1433  -15.0150  -14.8911  -14.8429  -14.7669  -14.7055
 -14.5918  -14.5598  -14.5058  -14.4551  -14.4207  -14.3274  -14.2530  -14.1452
 -14.0087  -13.9719  -13.8782  -13.7893  -13.6982  -13.4846  -13.3787  -13.3590
 -13.2516  -12.9351  -12.8084  -12.7629  -12.6828  -12.6243  -12.4207  -12.2828
 -12.2632  -12.1622  -12.1275  -12.0475  -11.9833  -11.9461  -11.9378  -11.8095
 -11.6790  -11.4965  -11.4592  -11.2679  -11.2070  -10.8094  -10.8058  -10.5315
 -10.4329  -10.2674   -9.7735   -8.8990   -8.4622   -6.3817   -0.0745    0.3277
   0.5806    0.9883    1.1081    1.1316    1.1684    1.5466    1.6618    1.9505
   1.9770    2.0143    2.0833    2.2907    2.4502    2.5490    2.5737    2.5877
   2.6078    2.6356    2.7685    2.8277    2.8570    2.9270    2.9866    3.1486
   3.2085    3.2217    3.2521    3.2880    3.3400    3.3716    3.3898    3.4466
   3.4869    3.5155    3.5377    3.5581    3.5800    3.6719    3.7026    3.7561
   3.7901    3.7982    3.9065    3.9232    3.9579    4.0335    4.0705    4.1618
   4.2641    4.3544    4.4321    4.4472    4.5084    4.5762    4.7001    4.7028
   4.7404    4.7679    4.8582    4.8932    5.0976    5.1610    5.2934    5.3782
   5.3946    5.4465    5.5534    5.6545    5.6717    5.6993    5.8210    5.8372
   5.8415    5.9644    6.0353    6.0752    6.0763    6.0792    6.1909    6.2552
   6.4892    6.6807    6.6861    6.9638    7.0345    7.1797    7.2617    7.9534


 Heat of formation   =       -853.97484026 kcal/mol  (      -37.03112789 eV)

 Total SCF energy    =    -240264.64731378 kcal/mol  (   -10418.65692354 eV)
 Electronic energy   =   -2253752.28613543 kcal/mol  (   -97730.03278849 eV)
 Core-core repulsion =    2013487.63882165 kcal/mol  (    87311.37586495 eV)
 
    Atomic Charges for Step       1 :
  Atom    Element       Mulliken Charge
     1      P                  2.788
     2      O                 -1.099
     3      O                 -0.835
     4      O                 -0.750
     5      C                  0.023
     6      C                 -0.033
     7      O                 -0.270
     8      C                  0.001
     9      O                 -0.308
    10      C                  0.004
    11      O                 -0.739
    12      C                  0.158
    13      N                 -0.178
    14      C                 -0.075
    15      N                 -0.193
    16      C                 -0.018
    17      C                 -0.457
    18      N                  0.095
    19      N                 -0.086
    20      C                 -0.134
    21      N                 -0.183
    22      C                 -0.122
    23      O                 -1.280
    24      P                  2.780
    25      O                 -1.090
    26      O                 -0.837
    27      O                 -0.740
    28      C                  0.016
    29      C                  0.002
    30      O                 -0.272
    31      C                  0.010
    32      O                 -0.313
    33      C                 -0.030
    34      O                 -0.321
    35      C                  0.102
    36      N                 -0.219
    37      C                  0.004
    38      C                 -0.216
    39      C                  0.352
    40      O                 -0.370
    41      N                 -0.438
    42      C                 -0.070
    43      C                 -0.192
    44      C                 -0.079
    45      P                  2.629
    46      O                 -1.062
    47      O                 -0.765
    48      O                 -0.822
    49      H                  0.108
    50      H                  0.113
    51      H                  0.143
    52      H                  0.145
    53      H                  0.161
    54      H                  0.141
    55      H                  0.214
    56      H                  0.174
    57      H                  0.125
    58      H                  0.084
    59      H                  0.159
    60      H                  0.114
    61      H                  0.110
    62      H                  0.134
    63      H                  0.147
    64      H                  0.117
    65      H                  0.155
    66      H                  0.143
    67      H                  0.214
    68      H                  0.226
    69      H                  0.220
    70      H                  0.237
    71      H                  0.222
    72      H                  0.227
    73      H                  0.221
    74      H                  0.298
    75      H                  0.177
    76      H                  0.143
    77      H                  0.291
    78      H                  0.091
    79      H                  0.287
    80      H                  0.289
 Total Mulliken Charge =       0.000
 
                  X        Y        Z     TOTAL  
  QM DIPOLE     1.354   -1.901    2.173    3.189
 
 Final Structure

  QMMM: QM Region Cartesian Coordinates (*=link atom) 
  QMMM: QM_NO.   MM_NO.  ATOM         X         Y         Z
  QMMM:     1        1      P       25.0351   21.7251   42.6089
  QMMM:     2        2      O       26.0514   20.7031   42.5049
  QMMM:     3        3      O       24.0762   21.7793   41.3453
  QMMM:     4        4      O       25.4471   23.2315   42.8817
  QMMM:     5        5      C       24.5718   24.2981   42.5736
  QMMM:     6        6      C       25.2977   25.2600   41.6285
  QMMM:     7        7      O       25.0967   24.8557   40.2692
  QMMM:     8        8      C       24.8237   26.7165   41.7877
  QMMM:     9        9      O       25.8849   27.5591   42.1684
  QMMM:    10       10      C       24.2031   27.1062   40.4239
  QMMM:    11       11      O       24.9666   28.1195   39.8218
  QMMM:    12       12      C       24.3302   25.8205   39.5356
  QMMM:    13       13      N       23.0624   25.2678   39.1370
  QMMM:    14       14      C       22.4966   24.0192   39.4398
  QMMM:    15       15      N       21.2732   23.8512   38.8831
  QMMM:    16       16      C       21.0256   25.0304   38.1826
  QMMM:    17       17      C       19.8882   25.4947   37.4363
  QMMM:    18       18      N       18.6889   24.6922   37.4414
  QMMM:    19       19      N       19.8332   26.8210   37.0221
  QMMM:    20       20      C       20.8931   27.5661   37.1767
  QMMM:    21       21      N       22.1821   27.2378   37.7610
  QMMM:    22       22      C       22.1275   25.9290   38.3146
  QMMM:    23       23      O       23.9814   21.4751   43.7992
  QMMM:    24       24      P       22.9602   21.2474   45.0197
  QMMM:    25       25      O       22.0622   22.3483   45.2771
  QMMM:    26       26      O       23.9747   20.9805   46.2169
  QMMM:    27       27      O       22.4009   19.7953   44.7053
  QMMM:    28       28      C       23.2841   18.6941   44.6516
  QMMM:    29       29      C       22.8889   17.6681   45.7210
  QMMM:    30       30      O       23.0535   16.3568   45.1736
  QMMM:    31       31      C       23.7464   17.7628   46.9994
  QMMM:    32       32      O       22.8586   17.6944   48.0961
  QMMM:    33       33      C       24.7266   16.5724   46.9105
  QMMM:    34       34      O       24.6798   15.8649   48.1318
  QMMM:    35       35      C       24.1868   15.6794   45.7486
  QMMM:    36       36      N       25.1938   15.3410   44.7559
  QMMM:    37       37      C       25.0090   14.1520   44.0341
  QMMM:    38       38      C       25.6484   13.8942   42.8661
  QMMM:    39       39      C       25.4721   12.6167   42.1334
  QMMM:    40       40      O       25.3922   12.5695   40.8883
  QMMM:    41       41      N       25.3858   11.4507   42.8646
  QMMM:    42       42      C       26.5342   14.8946   42.2055
  QMMM:    43       43      C       26.4981   16.2045   42.9014
  QMMM:    44       44      C       25.8501   16.3875   44.0685
  QMMM:    45       45      P       24.2649   29.5506   39.5634
  QMMM:    46       46      O       23.0943   29.8288   40.3705
  QMMM:    47       47      O       24.2985   29.6487   37.9824
  QMMM:    48       48      O       25.4992   30.5167   39.8484
  QMMM:    49       49      H       23.6301   23.9016   42.1082
  QMMM:    50       50      H       24.3450   24.8276   43.5421
  QMMM:    51       51      H       26.4092   25.1780   41.8018
  QMMM:    52       52      H       24.0655   26.8276   42.6112
  QMMM:    53       53      H       23.1361   27.4497   40.5468
  QMMM:    54       54      H       24.9376   26.0464   38.6062
  QMMM:    55       55      H       23.0283   23.2752   40.0466
  QMMM:    56       56      H       20.8388   28.6218   36.8526
  QMMM:    57       57      H       23.1614   18.2173   43.6369
  QMMM:    58       58      H       24.3404   19.0502   44.7875
  QMMM:    59       59      H       21.7926   17.7627   45.9618
  QMMM:    60       60      H       24.3161   18.7317   47.0524
  QMMM:    61       61      H       25.7709   16.9379   46.7172
  QMMM:    62       62      H       23.7678   14.7173   46.1738
  QMMM:    63       63      H       24.3200   13.4238   44.5019
  QMMM:    64       64      H       26.2183   15.0155   41.1311
  QMMM:    65       65      H       27.0314   17.0381   42.4217
  QMMM:    66       66      H       25.8147   17.3685   44.5739
  QMMM:    67       67      H       25.5961   11.4447   43.8280
  QMMM:    68       68      H       25.3360   10.5865   42.3851
  QMMM:    69       69      H       26.5170   27.5870   41.4355
  QMMM:    70       70      H       18.9774   23.7169   37.3444
  QMMM:    71       71      H       18.0863   24.9589   36.6625
  QMMM:    72       72      H       23.3592   17.3930   48.8663
  QMMM:    73       73      H       25.3897   15.2111   48.1051
  QMMM:    74       74      H       24.1559   21.0892   40.6902
  QMMM:    75       75      H       18.1646   24.7955   38.3171
  QMMM:    76       76      H       22.8833   27.2659   37.0306
  QMMM:    77       77      H       23.9928   21.6128   46.9309
  QMMM:    78       78      H       27.5915   14.5074   42.1913
  QMMM:    79       79      H       25.1420   29.8817   37.5955
  QMMM:    80       80      H       25.5367   30.9664   40.6878

           --------- Calculation Completed ----------

